### PR TITLE
Feature/73 activity detail todos

### LINKED ISSUE #73: Address ActivityDetail view TODOs
**Summary.**
In the app `activities` the `ActivityDetail` view is missing the implementation of the `post` method, as shown by the TODOs in the next fragment of code.

https://github.com/CapituloJaverianoACM/WebSiteAPI/blob/8e841028c2dda4bed5060c12fe841dd51a7500de/acm_web_site/apps/business/activities/views.py#L56-L60

### PR DESCRIPTION
**Pull request by BeltranDaniel**
### Summary
`ActivityDetail` view was refactored to allow insertion of many to many relationships between the Activity and Member classes.

### Test Plan
Github actions

### Related Issues
close #73

### CODE REVIEW

File: acm_web_site/apps/business/activities/serializers.py [hunk 1]
```diff
@@ -27,6 +28,7 @@ class ActivitySerializer(serializers.ModelSerializer):
     class Meta:
         model = Activity
         fields = '__all__'
+        depth = 1
```
Comment (camiloserr) [inline acm_web_site/apps/business/activities/serializers.py:31]: what is this for?

**Review by camiloserr — APPROVED** [review]
Grrrrrreat

File: acm_web_site/apps/business/activities/serializers.py (hunk 1 above)
Comment (BeltranDaniel) [reply acm_web_site/apps/business/activities/serializers.py:31]: This determines how many levels of depth are searched in the database. For example in the relationships Activity - ActivityMember - Member tables, having depth = 1 when retrieving Activity information shows also the complete information of the members associated as shown below. 

Captura de pantalla de 2020-07-29 17-01-52

When depth is not specified the same query returns the next results:
Captura de pantalla de 2020-07-29 17-07-22

**Review by DorianMoreno — APPROVED** [review]
Looking good!

**Pull request merged**